responsibleai/tests/rai_insights/test_rai_insights.py

```diff
@@ -52,7 +52,8 @@ class TestRAIInsights(object):
     def test_rai_insights_iris(self, manager_type):
         X_train, X_test, y_train, y_test, feature_names, classes = \
             create_iris_data()
-        models = create_models_classification(X_train, y_train)
+        X_train_dropped = X_train.drop(columns=['petal length'])
```

Comment:
nit: dropped indicates to me that this is what has been dropped, rather than what's remaining.

Reply:
update to `X_train_after_drop`